### ISSUE TITLE
SBGR Runway & Fix Locations

### ISSUE THREAD

**Issue by erikquinn** [zlsa/atc#399]
I'm not sure why, but it looks like the runways and fixes at SBGR are not in their correct positions. I noticed this when I went to add two new fixes (IG and PIER)... they were not where they I expected them to be, relative to the airport. Turns out the airport is actually placed 15 miles away from where it should be, and the fixes are shifted to match.

In the below picture, you can see this. I moved runway 9L-27R to the correct position, and also moved CGO to the correct position (old one is still there, labeled "CGO-OLD" for demonstration purposes).

![image](https://cloud.githubusercontent.com/assets/5103735/13188432/90419196-d71f-11e5-8658-e60c795ca383.png)

To make future elaborations on this airport easier, this should be corrected to use the real-world positions.

**Comment by MaicomMR** [comment]
Your runway positions is in the wrong position.

This is the airport view:
![untitled-1](http://photos.airliners.net/aviation-photos/photos/6/3/0/0988036.jpg)

I'm live on Brazil and fly to São Paulo some times, I make this summary of city airports.
![untitled-1](https://cloud.githubusercontent.com/assets/12571488/13191365/5032ad6c-d74a-11e5-957b-98f11d9a94d1.png)
SBGR > GRU Airport is the international Airport
SBMT > Campo de Marte - small planes, private and police helicopters base
SBSP > National Airport - Flights to all brazil, its the Urban Carrier, only small planes can land on them(A318, A319, B738)
The CGO waypoint is above the SBSP airport.

When I rebuild the SBGR I use official AIS(Aeronautical Information System,) charts, with real runaways, waypoints and SID coordinates, but someone change the airport after.

You can found official charts here:
http://www.aisweb.aer.mil.br/?i=cartas&filtro=1&nova=1

Sorry my bad english.

**Comment by erikquinn** [comment]
@MaicomMR You are right... they _were_ correct, until commit cd6c496, when they were changed to the positions in the bottom picture (middle of the forest). I'm not sure why this was done, but since all of the other fixes were moved as well, I'm assuming that was meant to move them to the new airport position. Strange.

What it used to be, when @MaicomMR first added it:
![image](https://cloud.githubusercontent.com/assets/5103735/13192456/2277a2f2-d73a-11e5-8a27-cd144b281fd2.png)

What it changed to:
![image](https://cloud.githubusercontent.com/assets/5103735/13192461/32ce72a2-d73a-11e5-84e4-5197d140cfe8.png)

Also, thanks for the AIS link. Very helpful to have current charts. :smile:

**Comment by MaicomMR** [comment]
As I said, when I create I use the official coordinates, but @glangford  change the airport and merge it :confused: How do you see the game coordinates on map? thanks.

Your welcome, AIS is the official site of Brazilian Real Aviation Charts and air meteorology, enjoy.

**Comment by MaicomMR** [comment]
Hey, I'm searching my files and i found the problam, @glangford help me to put the new code with the SIDs, I build this wrong version, not glan, look:

https://github.com/zlsa/atc/issues/234

**Comment by MaicomMR** [comment]
Finnaly i found the problem, the most improbable hapening, the official coordinates are wrong. or the google maps are wrong, look this oficial coordinates in google maps:
![untitled-1](https://cloud.githubusercontent.com/assets/12571488/13194385/5c3cb3f0-d777-11e5-9849-41071c5cc84a.png)

And here is the official document with the coordinates:
http://www.aisweb.aer.mil.br/download/?tabela=f14ca80b-29bd-4936-843de6c39d2f9142

I built this coordinates to build a SID version of the GRU airport, the older version I use only google maps to take the runways start/end.

I will study more about this to try slove faster.

**Comment by erikquinn** [comment]
@MaicomMR Huh! That's surprising.

Actually though, I think I've got it all fixed now. I used the latest charts to implement the various altitudes, speeds, and runways allowed for each one. I just now finished up at SBGR. Seeing that you are the local expert on SBGR, could you have a look and let me know about anything that's not working right, particularly with the SIDs? I have added:
- `CGO.NIBGA (1B)`, 
- `CGO.LOLAG (1B)`
- `CGO.OBMAV (1B)`
- `CGO.RCL (1B)`
- `ROMIB.DORLU (1B)`
- `ROMIB.EKIDI (1B)`
- `ROMIB.UBRIR (1B)`
- `BGC.UKBEV (2A/3A/1A)`
- `BGC.MULAP (2A/3A/1A)`
- `DORLU.DORLU (1A)`
- `IVRAG.VUMEV (1A)`
- `IVRAG.UBRIR (1A)`
- `SCP.DAGUB (1A)`
- `SCP.UBRIR (1A)`
- `SCP.USAMO (1A)`
- `SCP.VUMEV (1A)`

As a reminder, you can use `caf` ("cleared as filed") to clear an aircraft onto its filed route, without needing to assign it a sid. You can also use `cvs` ("climb via sid") to tell the a/c to climb in accordance with the charted speeds and altitudes.

[erikquinn.github.io/atc](http://erikquinn.github.io/atc)

**Comment by MaicomMR** [comment]
Cool! I'm building a map with coordinates and map here, I will share with you at ready. here is the preview:
![untitled-2](https://cloud.githubusercontent.com/assets/12571488/13194622/e2dc207e-d77e-11e5-9bea-155053b6bb4f.png)
I will export at finish.

**Comment by MaicomMR** [comment]
Ops, sorry, i will check your version now :smiley:

**Comment by MaicomMR** [comment]
Wow, your Departure system souds fanstastic, but i missed approach waypoints, I will work on this.

And here are the points I picked up so far.

ILS S
S 23:27:07.26,W 46:42:32.64 VUSAG
S 23:28:51.53,W 46:38:37.05 ISKUL
S 23:27:44.10,W 46:34:26.45 GR036
S 23:28:02.40,W 46:18:02.40 RAFA

VOR RWY 27L
S 23:16:25.75, W 46:16:30.86 ANSUG
S 23:22:39.00, W 46:16:24.60 EDPAM
S 23:20:56.81, W 46:10:09.57 VURAK
S 23:24:23.14, W 46:23:07.67 BCO
S 23:25:55.81W 46:27:30.59 FTP27L
S 23:28:02.40,W 46:18:02.40 RAFA

STAR RNAV URDOSO 1D RWY09L/09R
S 23:58:59.33, W 46:15:48.02 SAT
S 24:36:07.80, W 46:37:31.20 ANISE
S 24:17:31.20, W 46:26:36.60 SIPUK
S 24:03:24.77, W 46:18:22.49 TIVNA
S 23:53:58.38, W 46:15:53.09 GR001
S 23:44:32.60, W 46:16:02.61 GR003
S 23:50:56.61, W 45:38:41.27 VUKIK
S 23:55:01.80, W 45:57:25.20 TOGES
S 23:37:10.36, W 45:41:24.07 PAGOG
S 23:39:31.64, W 46:16:07.67 GR004
S 23:32:42.34, W 46:16:14.53 GR006
S 23:16:25.75, W 46:16:30.86 ANSUG
S 23:20:25.53, W 46:29:58.10 KOVLA
S 23:23:04.74, W 46:40:26.53 UTBUR
S 23:19:10.14, W 46:49:15.48 UROSO
S 23:23:12.54, W 46:51:21.80 GR703
S 23:24:30.21, W 46:48:25.84 GR602
S 23:25:49.07, W 46:45:29.08 GR401
S 23:27:07.26, W 46:42:32.64 VUSAG

STAR RNAV ANSUG 1B RWY 09L/09R
S 23:02:44.29, W 45:31:00.15 TBE
S 23:07:06.47, W 45:45:25.95 MAKVA
S 23:10:12.93, W 45:55:44.95 VAPEM
S 23:16:25.75, W 46:16:30.86 ANSUG
S 22:24:46.57, W 46:17:20.22 MOXEP
S 22:41:43.96, W 46:17:03.78 DIGAX
S 22:51:27.89, W 46:16:54.32 ROKUM
S 22:57:49.80, W 46:16:48.00 IBDIK
S 23:09:04.00, W 46:16:37.14 SUKNO
S 23:18:53.33, W 46:24:47.16 PAGAL
S 23:20:25.53, W 46:29:58.10 KOVLA
S 23:23:47.37, W 46:41:21.28 MARPU

**Comment by erikquinn** [comment]
@MaicomMR thanks! I now added all those waypoints, and pushed them live at [http://erikquinn.github.io/atc](http://erikquinn.github.io/atc). :+1: If you have more points to add, please list only the new ones (it will make it easier for me). :smile:

**Comment by MaicomMR** [comment]
@erikquinn one sugestion to your version, move the Runway numbers, I cant see the numbers because they are very close.

I do this test and sounds good:

  "runways": [
    {
      "name": ["9L", "27R"],
      "name_offset": [[0,1], [1.5 , 1]],
      "end": [["S23d26m04.24", "W46d29m03.54"], ["S23d25m28.94", "W46d26m52.79"]],
      "length": 3.700,
      "delay": [10, 50],
      "ils": [true, true]
    },
    {
      "name": ["9R", "27L"],
      "name_offset": [[0, -1], [0, -1]],
      "end": [["S23d26m20.07", "W46d29m14.82"], ["S23d25m52.35", "W46d27m32.18"]],
      "length": 2.900,
      "delay": [15, 60],
      "ils": [true, false]
    }
  ],

**Comment by erikquinn** [comment]
@MaicomMR Good idea. They look better now.
![image](https://cloud.githubusercontent.com/assets/5103735/13195118/9adff81a-d777-11e5-81aa-5c73dfd4fe4f.png)

**Comment by MaicomMR** [comment]
Cool! Now its better to select a runway to land and one to takeoffs :+1:

**Comment by MaicomMR** [comment]
More two waypoints from ILS approach (09 runways)

```
 "IKSUL" : ["S23d27.90m53", "W46d38.37m05"],
 "SMR" : ["S23d31.20m53", "W46d50.37m05"]
```

**Comment by MaicomMR** [comment]
Now I'm working on some restricted zones in your version of SBGR to be compatible on the future merge.

Real Charts:
![Uploading aeroboero-AB11-4.jpg…](http://puu.sh/neDOs/8233079f1e.jpg)
ATC Screen:
![Uploading aeroboero-AB11-4.jpg…](http://puu.sh/neDSH/504b9812cb.jpg)

**Comment by erikquinn** [comment]
@MaicomMR added those waypoints.

**Comment by MaicomMR** [comment]
I test for a long period in high speed(2x) and its fine! no crashes, and all airplanes land in a cool queue, your system of departure is really fantastic. :+1: 

![ss 2016-02-20 at 08 44 40](https://cloud.githubusercontent.com/assets/12571488/13196194/ab34f8f0-d7ae-11e5-98e3-6b9135d191b1.jpg)

**Comment by erikquinn** [comment]
Resolved by #404.
